bleak/backends/bluezdbus/scanner.py

```diff
@@ -304,4 +303,38 @@ def parse_msg(self, message):
         )
 
         if self._callback is not None:
-            self._callback(message)
+            # First ensure that the message body is a list and has the dict where we can find the information we need
+            if isinstance(message.body, list) and len(message.body) > 2:
```

Comment:
When would this be false? Are there other kinds of messages that are received?

If it is not expected to ever be false, we should probably remove the check. Otherwise, it would be nice to invert the check so that the error is handled in the indented if statement and the normal code flow is not indented.

Reply:
I added the check to be safe with accessing the list in the next line, I haven't seen a case yet where it didn't pass so it should be safe to remove.